src/courses/beginner/03.md

```diff
@@ -85,17 +87,13 @@ control 'tmp-1.0' do                        # A unique ID for this control
 end
 ```
 
-::: tip Tip for developing profiles
-When creating new profiles use the existing example file as a template
-:::
-
 This example shows two tests. Both tests check for the existence of the `/tmp` directory. The second test provides additional information about the test. Let's break down each component.
 
-- `control` (line 9) is followed by the control's name. Each control in a profile has a unique name.
-- `impact` (line 10) measures the relative importance of the test and must be a value between 0.0 and 1.0.
-- `title` (line 11) defines the control's purpose.
-- `desc` (line 12) provides a more complete description of what the control checks for.
-- `describe` (lines 13 — 15) defines the test. Here, the test checks for the existence of the `/tmp` directory.
+- `control` (line 10) is followed by the control's name. Each control in a profile has a unique name.
+- `impact` (line 12) measures the relative importance of the test and must be a value between 0.0 and 1.0.
+- `title` (line 13) defines the control's purpose.
+- `desc` (line 14) provides a more complete description of what the control checks for.
+- `describe` (lines 15 — 17) defines the test. Here, the test checks for the existence of the `/tmp` directory.
 
 In Ruby, the `do` and `end` keywords define a `block`. An InSpec control always contains at least one `describe` block. However, a control can contain many `describe` blocks.
 ::: tip More information on a block in Ruby
```

Comment:
This link is insecure - maybe we use [this](https://www.rubyguides.com/2016/02/ruby-procs-and-lambdas/) instead?